back/src/domains/convention/adapters/PgConventionQueries.integration.test.ts

```diff
@@ -575,23 +577,40 @@ describe("Pg implementation of ConventionQueries", () => {
         ].map((params) => conventionRepo.save(params)),
       );
 
-      const createNewEvent = makeCreateNewEvent({
-        timeGateway: new RealTimeGateway(),
-        uuidGenerator: new UuidV4Generator(),
-      });
-      const eventEmailSentToImmersion1 = createNewEvent({
-        topic: "EmailWithLinkToCreateAssessmentSent",
-        payload: {
-          id: validatedImmersionEndingThe15thThatAlreadyReceivedAnEmail.id,
+      const notification1: Notification = {
```

Comment:
suggestion de nommage (car on n'utilise qu'une seule notification):

```suggestion
      const notification: Notification = {
```